pkgs/development/python-modules/ipymarkup/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, intervaltree
+}: buildPythonPackage rec {
+  pname = "ipymarkup";
+  version = "0.9.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-P0v6EP1mKTIBr4SEp+a8tyO/LjPZpqoAiCZxr5yiaRE=";
+  };
+
+  propagatedBuildInputs = [ intervaltree ];
+  pythonImportCheck = [ "ipymarkup" ];
```

```suggestion
  pythonImportCheck = [ "ipymarkup" ];
  
  # upstream has no tests
  doCheck = false;
```
Would you please add `doCheck = false`?

---

I've added this change into `ipymarkup` commit.